go-controller/pkg/util/multi_network.go

```diff
@@ -761,6 +761,21 @@ func ValidateNetConf(nadName string, netconf *ovncnitypes.NetConf) error {
 		return fmt.Errorf("the subnet attribute must be defined for layer2 primary user defined networks")
 	}
 
+	//ensure that the cluster supports ipfamilies that the NAD is using
```

Comment:
Please note that `parseSubents` is executed as part of `ParseNadInfo` by  `NewNetInfo`.
Maybe it"ll fit better have this check in `NewNetInfo` and save additional parsing.

Reply:
moved the code good suggestion